db/kv_dense_block.go

```diff
@@ -58,6 +58,13 @@ func NewDenseBlock(key string, startClock base.PartitionClock) *DenseBlock {
 	}
 }
 
+func (d DenseBlock) String() string {
+	return fmt.Sprintf("key: %s, count: %d",
+		d.Key,
+		d.Count(),
```

Comment:
Attempting to log a non-initialized block as string - i.e. len(value) = 0 - is going to panic on the call to getEntryCount().  We might want to add some slice safety to getEntryCount if we're going to include Count() in string (i.e. return count = zero if len(value) < 2).

Reply:
Fixed.  I also thought it might be worth exercising an uninitialized `DenseBlock{}` since that's a valid use case, and so I updated one of the tests.